utils/utils.tsx

```diff
@@ -123,26 +134,18 @@ export function isKeyPressed(event, key) {
     return event.keyCode === key[1];
 }
 
-/**
- * check keydown event for line break combo. Should catch alt/option + enter not all browsers except Safari
- * @param  {object}  e - keydown event
- * @return {boolean}
- */
-export function isUnhandledLineBreakKeyCombo(e) {
+// check keydown event for line break combo. Should catch alt/option + enter not all browsers except Safari
```

Comment:
We should actually continue wrapping these with `/** */` so that it's a JSDoc comment. If we do that, it'll show a tooltip in VS Code (and presumably other editors)

From the current version of the file:
![Screen Shot 2022-04-27 at 11 05 13 AM](https://user-images.githubusercontent.com/3277310/165549954-5de7a00e-c729-402c-a91f-32ac83dab7db.png)
![Screen Shot 2022-04-27 at 11 05 21 AM](https://user-images.githubusercontent.com/3277310/165549968-d10a7dd9-2bdc-4973-84d0-cd2ca41f5625.png)

That said, I'm okay with removing the parameters and return value documentation. Those should usually be straightforward enough that I've rarely felt the need to document them myself.